### PR TITLE
INTG-2663 - Configuration Page Updates

### PR DESCRIPTION
**Pull request by sc-gv**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -31,10 +31,9 @@ func main() {
 
 	// Create application with options
 	err = wails.Run(&options.App{
-		Title:         "SafetyCulture Exporter",
-		Width:         1024,
-		Height:        768,
-		DisableResize: true,
```
Comment (MickStanciu) [inline main.go:37]: is everything working ok after this? I mean: if somebody makes the window smaller?

File: frontend/src/components/Export/Pill.svelte [hunk 2]
```diff
@@ -42,7 +42,7 @@
     }
 
     .error {
-        background-color: #c2586c;
-        color: #b50d0d;
+        background-color: #A9282E;
+        color: #FFF0F1;
```
Comment (MickStanciu) [inline frontend/src/components/Export/Pill.svelte:46]: pretty sure this change was made in [INTG-2665](https://safetyculture.atlassian.net/browse/INTG-2665)

File: scripts/build-linux.sh [hunk 3]
```diff
@@ -4,4 +4,6 @@ echo "BUILDING WITH VERSION: ${VERSION}"
 
 wails build -platform linux/amd64 -clean -ldflags "-s -w -X github.com/SafetyCulture/safetyculture-exporter-ui/internal/version.version=${VERSION}"
 
-tar -czf exporter-linux-amd64.tar.gz ./build/bin/safetyculture-exporter
+mv ./build/bin/safetyculture-exporter .
+
+mv safetyculture-exporter exporter-linux-amd64
```
Comment (MickStanciu) [inline scripts/build-linux.sh:9]: what was the problem here?

**Review by MickStanciu — COMMENTED** [review]
Left some questions

File: scripts/build-linux.sh (hunk 3 above)
Comment (sc-gv) [reply scripts/build-linux.sh:9]: No problem. We are not zipping the executables.

File: main.go (hunk 1 above)
Comment (sc-gv) [reply main.go:37]: Yeah looks okay unless we make the window really small. 


https://user-images.githubusercontent.com/92243746/218865812-5265b8ea-eeb6-4861-abd0-49fb06286bee.mov

Comment (MickStanciu) [reply main.go:37]: is Akash and Austin ok with this change?

File: scripts/build-linux.sh (hunk 3 above)
Comment (MickStanciu) [reply scripts/build-linux.sh:9]: why not zipping the executables? Or is less friction for the user?

File: main.go (hunk 1 above)
Comment (MickStanciu) [reply main.go:37]: My reason can be the fact a resizing window will loose the feeling of an app. If Austin is ok,.. I have no objections

File: scripts/build-linux.sh (hunk 3 above)
Comment (sc-gv) [reply scripts/build-linux.sh:9]: Yes. less friction for the user.

Comment (MickStanciu) [reply scripts/build-linux.sh:9]: Had a chat with Austin and not zip is fine

**Review by MickStanciu — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (sc-gv) [reply main.go:37]: Done.

**Pull request merged**